tools/analyzer_plugin/lib/src/util/ast_util.dart

```diff
@@ -3,15 +3,12 @@ library over_react_analyzer_plugin.src.ast_util;
 
 import 'dart:collection';
 import 'dart:mirrors';
-// This is necessary for `ConstantEvaluator`. If that API is removed, it can just
-// be copied and pasted into this analyzer package (if still needed).
-// ignore: deprecated_member_use
-import 'package:analyzer/analyzer.dart' show ConstantEvaluator;
```

Comment:
Oh, looks like this answers my previous question. Did they replace this with an alternative or a better option?